synapse/models/transport.py

```diff
@@ -52,11 +106,169 @@ def getModelDefs(self):
                 ('transport:sea:telem', ('guid', {}), {
                     'doc': 'A telemetry sample from a vessel in transit.'}),
 
-                # TODO a few more items to plumb eventually
-                # ('transport:sea:hin',
-                # ('transport:sea:port',
+                ('transport:rail:train', ('guid', {}), {
```

Comment:
How does this differ from other land-based transport where we need a special form for trains? Currently transport:land:segment is an individual segment. Other land-based transport (besides trains) may include multiple segments / stops (e.g., a bus route, or a standard long-haul trucking route, or even a local FedEx delivery route).

Would a more generic transport:land:route be a better fit for "land transport" that commonly includes multiple segments?

I also found the 'transport:rail:train' name a bit confusing - as the model expands, the naming conventions within transport:* aren't always intuitive as to which forms are "objects" (planes, containers) and which are "instances of moving from one place to the next". tl;dr transport:rail:train sounds like it should be a vehicle/object.